lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -360,16 +404,20 @@ def _delete_nodes(
             except Exception as ex:
                 log.warning(f"console log delete failed. {ex}")
 
-            # Delete the files created for the VM.
-            try:
-                os.remove(node_context.os_disk_file_path)
-            except Exception as ex:
-                log.warning(f"OS disk delete failed. {ex}")
+            if self.qemu_platform_runbook.is_host_remote():
+                vm_dir = os.path.dirname(node_context.os_disk_file_path)
```

Comment:
`_delete_nodes` shouldn't be generating any values. All the values required by `_delete_nodes` should be generated in `_configure_nodes` and then stored.

Reply:
This doesn't _generate_ any values. It is simply _extracting_ the path to the parent folder from the values already stored in `node_context`.